style.css

```diff
@@ -0,0 +1,79 @@
+.header-container {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 718px;
+  margin-left: 20px;
+  background-image: url('/images/header-shapes mobile@2x.png');
```

Comment:
Great job using CSS to style your background image👍. However, your image styling is currently not showing in the browser, as seen in the screenshot below:

<img width="237" alt="image" src="https://user-images.githubusercontent.com/37116322/219205544-a361303e-191c-4bef-98c4-af5d2ea85c67.png">

I believe the issue is related to the path for your images. Make sure to start each image path with a `.` so that the application can properly find each file. e.g `'./images/header-shapes mobile@2x.png'`. This error is affecting all your images so please take a look at each of them🙏